modules/rostests/apitests/powrprof/power.c

```diff
@@ -14,6 +14,7 @@
 #include <ntstatus.h>
 #define WIN32_NO_STATUS
 #include <winreg.h>
+#include <potypes.h>
```

Comment:
> SYSTEM_POWER_INFORMATION does belong in ntpoapi only, which means if you need it in user mode, it must go in ndk/potypes.h enclosed in a NTOS_MODE_USER ifdef. 

@ThFabba, should I even simply copy the definition here, as I assume it should be needed in this apitest only?

Reply:
No, NDK is the right place.
Can you include this as `<ndk/potypes.h>` please though?

Reply:
Done.